layout/common/scripts.jsx

```diff
@@ -1,6 +1,7 @@
 const { Component, Fragment } = require('inferno');
 const { toMomentLocale } = require('hexo/dist/plugins/helper/date');
 const Plugins = require('./plugins');
+const pjaxMinJsPath = require.resolve('pjax/pjax.min.js');
```

Comment:
Why not use the `pjax` from CDN but the `pjax` from `node_modules`? Will this file even present in the generated `public/` directory when you publish your site?

Reply:
I'm a bit confused about how inferno components work here.

Will it be better if I make pjax a plugin?
`layout/plugin/pjax.jsx`:
```javascript layout/plugin/pjax.jsx
const { Component, Fragment } = require('inferno');
const { cacheComponent } = require('hexo-component-inferno/lib/util/cache');

class Pjax extends Component {
    render() {
        return <Fragment>
            <script src={cdn('pjax', '0.2.8', 'min/pjax.min.js')}></script>
            <script src={this.props.jsUrl}></script>
        </Fragment>;
    }
}

Pjax.Cacheable = cacheComponent(Pjax, 'plugin.Pjax', props => {
    const { helper, head } = props;
    return {
        head,
        jsUrl: helper.url_for('/js/pjax.js')
    };
});

module.exports = Pjax;
```